public/config/configuration.json

```diff
@@ -8,6 +8,7 @@
   "BUSINESS_CREATE_LD_CLIENT_ID": "5e6142dd29b3030a850285b3",
   "DASHBOARD_URL": "https://dev.bcregistry.ca/business/",
   "HOTJAR_ID": "",
+  "IA_SURVEY_ID": "",
```

Comment:
When configured, checks whether to display Filing Survey dialog. Not configured when running locally.